### ISSUE TITLE
Animation-free UI

### ISSUE THREAD

**Issue by GreenLunar** [QupZilla/qupzilla#1894]
Offer an option to disable any smoothing UI animation, such as tab scrolling, collapse & expand, and opening of Find bar.

Much to my delight Otter Browser which is also built upon Qt5 does not have animated UI elements.

**Comment by nowrep** [comment]
Does it really bother you that much? There are only few animations and the animations in tabbar even make it easier to work with.

**Comment by GreenLunar** [comment]
> Does it really bother you that much?

Yes, it does, but not as much as other web browsers do. I despise animations.

> There are only few animations and the animations in tabbar even make it easier to work with.

Yes, but FF provides an option to disable animated tab-bar with `browser.tabs.animate`, yet it does not allow to disable find-bar animation in a similarly to `browser.fullscreen.animateUp` when is set to `0`; and in recent year it has removed an option to disable tab-bar animation when dragging tabs.

**Comment by GreenLunar** [comment]
Bug 783282 - When dragging a tab within the tab strip, move it directly instead of displaying a drop indicator
http://forums.mozillazine.org/viewtopic.php?f=38&t=2620001
https://bugzilla.mozilla.org/show_bug.cgi?id=783282

Bozi7

> So after updating to Firefox 17 I noticed that when you move a tab around it does a "slide" animation which is something new. I couldn't find anything about it in about:config, is there a way to disable it?

c-yan

> I want to disable it, too.
This effect make difficult to move to sub tree.

Bluefang

> **There is no option to disable it**. The tab drag-&-drop implementation was completely rewritten.
You'd need an extension to undo the changes.

**Comment by Martii** [comment]
This would probably be very useful for remote sessions I think to reduce potential flicker. Just got `$ sudo pacman -S tigervnc` *(client and server)* working here between the distros and disabled most of Plasma's effects in Settings.

**Comment by GreenLunar** [comment]
@Martii, may you check whether scrolling via a remote session using a touch pad or touch screen is always smooth (even when smooth scrolling is disabled)? See #1907

**Comment by Martii** [comment]
Without steering this topic too far off-track I prefer desktops over portable devices... but bringing it back on topic...

My "idiot" phone isn't all that "smart" so you would need to ask someone with an actual touch screen with RDC/VNC app to a RDP/VNCS... I'm not even sure if Android has anything like that. When I had my iPhone there was nothing in the available repos like that either. FxOS is pretty much "dead" e.g. on hold... so that leaves someone with a laptop that's touch screen enabled... don't have one of those.

Animations clog up networks and when on a metered plan may incur additional charges for what some consider useless animations... looks pretty though with animations though.

**Comment by ke7ofi** [comment]
@Martii but they only charge for animations that have to be downloaded...

**Comment by smarquespt** [comment]
I side note on this issue. In 2.0.0 there is no option to enable tab preview. Was this on qupzilla side or is it due QtWebEngine?
